### ISSUE TITLE
Issue with hover states on several links

### ISSUE THREAD

**Issue by hursey013** [GSA/digitalgov-pra#266]
The link in the `Official site...` banner, and the large button on the homepage both switch to a color without significant color contrast on hover.

![Screen Shot 2019-06-24 at 1 53 26 PM](https://user-images.githubusercontent.com/1178494/60040777-be5c9280-9687-11e9-8086-2955713f9d5c.png)

![Screen Shot 2019-06-24 at 1 53 37 PM](https://user-images.githubusercontent.com/1178494/60040781-c1f01980-9687-11e9-893b-a9e10d670ce3.png)

**Comment by hursey013** [comment]
Oops, looks like the top banner is resolved here https://github.com/GSA/digitalgov-pra/issues/227, but the `Learn what the PRA does` button looks like it still needs a fix.

**Comment by cmajel** [comment]
@hursey013 @jeremyzilar yea, I noticed this today.

Below is what we had originally planned for hover-styles for primary buttons:
<img width="596" alt="Screen Shot 2019-07-03 at 12 24 17 PM" src="https://user-images.githubusercontent.com/2374206/60608625-e3d75380-9d8d-11e9-88a4-718837615235.png">

We should do the background color shift but not change the text color to `white`. Changing it to white means the button does not have a high enough color contrast to be accessible.
